examples/openvino/image-classification/README.md

```diff
@@ -39,4 +39,48 @@ python run_image_classification.py \
     --save_total_limit 3 \
     --seed 1337 \
     --output_dir /tmp/beans_outputs/
-```
\ No newline at end of file
+```
+
+On a single V100 GPU, this example takes about 1 minute and yields a quantized model with accuracy of **98.5%**.
+
+### Joint Pruning, Quantization and Distillation (JPQD) of Swin on food101
+
+`OVTrainer` also provides advanced optimization workflow via NNCF to structurally prune, quantize and distill. Following is an example of joint pruning, quantization and distillation on Swin-base model for food101 dataset. To enable JPQD optimization, use an alternative configuration specified with `--nncf_compression_config`. For more details on how to configure the pruning algorithm, see NNCF documentation [here](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/experimental/torch/sparsity/movement/MovementSparsity.md).
+
+To run the JPQD example, please install optimum-intel from source. This command will install or upgrade optimum-intel and all necessary dependencies:
+
+```bash
+python -m pip install --upgrade "git+https://github.com/huggingface/optimum-intel.git#egg=optimum-intel[openvino,nncf]"
+```
```

Comment:
```suggestion
```
Same as for audio-classification, should no longer be necessary.

Reply:
Deleted. 
https://github.com/huggingface/optimum-intel/blob/bdb501859a81eed71733910b9aeedebe034b07dd/examples/openvino/image-classification/README.md
https://github.com/huggingface/optimum-intel/blob/bdb501859a81eed71733910b9aeedebe034b07dd/examples/openvino/audio-classification/README.md